packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1753,6 +1759,7 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
           child: widget.drawer,
           drawerCallback: _drawerOpenedCallback,
           dragStartBehavior: widget.drawerDragStartBehavior,
+          scrimColor: widget.scrimColor,
```

Comment:
Please add a test for this in `scaffold_test.dart`

Reply:
@rami-a Could you help me with this? What type of test should I do? Any test in scaffold_test.dart file that I could use as guide ?

Reply:
Sure, on second thought, this test probably belongs in `drawer_test.dart`. I would start the test similar to the one called 'Drawer control test` where you can set up a MaterialApp (but make sure to set the scrim color to something custom) and then open the drawer and pump the tester. 

Then the last part of the test would require digging through the widget tree to find the `Container` that is actually the scrim and then check that its color is correct. There are a few ways to do this but you can take a look at `_getSnackBarMaterial` inside `snack_bar_theme_test.dart` as an example of looking for a specific widget. 

Hope that helps!

Reply:
great! I added the test inside drawer_test.dart  :)   @rami-a